build.gradle

```diff
@@ -529,7 +529,10 @@ project(':iceberg-aws') {
     inputSpec.set(s3SignerSpec)
     recommend.set(true)
   }
-  check.dependsOn('validateS3SignerSpec')
+  tasks.named('check') {
```

Comment:
I would prefer to not change this line and keep it as is. This is so that we're in-line with how we define this in other places. And then right after the `integrationTest` task (after L526) is defined you would add the `check.dependsOn integrationTest`